en/components/grids_templates/export-excel.md

```diff
@@ -181,6 +181,18 @@ It is now possible to export @@igComponent with defined [multi-column headers](m
 </code-view>
 }
 
+## Export Grid with frozen column headers
+
+By default Excel Exporter service exports the Grid without pinned column headers. There are scenarios in which you may want to freeze all headers on top of the exported excel file as the user scrolls through the Grid. To achieve this you could set the [exporter option]({environment:angularApiUrl}/classes/igxexporteroptionsbase.html) [freezeHeaders]({environment:angularApiUrl}/classes/igxexporteroptionsbase.html#freezeHeaders) to `true`.
```

Comment:
Just a few tweaks - I think it sounds better but I am open for other suggestions:
```suggestion
By default Excel Exporter service exports the grid with scrollable (unfrozen) column headers. There are scenarios in which you may want to freeze all headers on top of the exported excel file so they always stay in view as the user scrolls through the records. To achieve this you could set the [exporter option]({environment:angularApiUrl}/classes/igxexporteroptionsbase.html) [freezeHeaders]({environment:angularApiUrl}/classes/igxexporteroptionsbase.html#freezeHeaders) to `true`.
```